dev/reference/system-databases/information-schema.md

```diff
@@ -2,6 +2,7 @@
 title: Information Schema
 summary: Learn how to use Information Schema in TiDB.
 category: reference
+aliases: ['/docs/sql/information-schema/'] 
```

Comment:
Be sure to remove this line from the `/dev` version.

Reply:
@morgo Thanks!